massgov/mayflower
# React DP-11634: MainNav a11y

## Description
Update the MainNav molecule for better a11y, enabling users to tab through the navigation menu. Also updated the svg-sprite-loader package to permit Icon component

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-11634)

## Steps to Test
1. `cd react`
2. `npm install`
3. `npm start`
4. go to http://localhost:6006/iframe.html?selectedKind=organisms&selectedStory=Header and confirm that you can tab through and on enter expand main nav like video below.

## Screenshots
![main-nav](https://user-images.githubusercontent.com/9359261/50616215-808b6800-0eb5-11e9-99ad-8e681303b555.gif)
